### PR TITLE
 Support httpapi in resource modules, part 4

### PR DESCRIPTION
**Pull request by Qalthos**
##### SUMMARY

Modifies the tests for several modules

See also #28

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

eos_static_routes
eos_vlans

### CODE REVIEW

File: tests/integration/targets/eos_static_routes/tests/common/_remove_config.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+- name: Setup
+  become: true
+
+  arista.eos.eos_config:
+    lines:
+      - "no vrf definition testvrf"
+      - "no vrf definition vrftest1"
```
Comment (GomathiselviS) [inline tests/integration/targets/eos_static_routes/tests/common/_remove_config.yaml:8]:
```suggestion
    
```

File: tests/integration/targets/eos_static_routes/tests/common/_remove_config.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      - "no ip route 10.1.1.0/24 Ethernet1 20.1.1.3 track bfd 200"
+      - "no ip route 10.50.0.0/16 Management1"
+      - "no ip route 23.1.0.0/16 Nexthop-Group testgrp tag 42"
+      - "no ip route 155.55.1.0/24 Nexthop-Group testgrp tag 100"
+      - "no ip route 122.1.19.0/24 Nexthop-Group testgrp 21"
+      - "no ip route vrf testvrf 120.1.1.0/24 Ethernet1 23"
+      - "no ip route vrf vrftest1 77.77.1.0/24 33.1.1.1"
+      - "no ipv6 route 1000:10::/64 Ethernet1 67 tag 98"
+      - "no ipv6 route vrf testvrf 1120:10::/64 Ethernet1 55"
+      - "no ipv6 route 1000:10::/64 Ethernet1 55"
```
Comment (GomathiselviS) [inline tests/integration/targets/eos_static_routes/tests/common/_remove_config.yaml:19]:
```suggestion
      - "no ipv6 route 1000:10::/64 Ethernet1 55"
      - "no vrf definition testvrf"
      - "no vrf definition vrftest1"
      
```

Comment (Qalthos) [reply tests/integration/targets/eos_static_routes/tests/common/_remove_config.yaml:16]: This didn't quite do the trick, but moving them to a separate call with `ignore_errors` did.

**Comment by Qalthos** [comment]
recheck

**Pull request merged**